odoo/addons/base/data/res_users_demo.xml

```diff
@@ -72,7 +72,7 @@
             <field name="zip">07002</field>
             <field name="country_id" ref="base.us"/>
             <field name="company_name">YourCompany</field>
-            <field name="image_1920" type="base64" file="base/static/img/partner_demo_portal.jpg"/>
+            <field name="image_1920" type="base64" file="base/static/img/partner_demo_portal.png"/>
```

Comment:
@edi-odoo , since we're adding a 128px image, is it "normal" that the field name is `image_1920` ? Could it be the reason for the issue we had with the 512px version?